README.md

```diff
@@ -95,6 +95,23 @@ redis-cli PING  # redis-cli can be installed with "apt install -y redis-tools"
 
 *You need `--ulimit memlock=-1` because some Linux distros configure the default memlock limit for containers as 64m and Dragonfly requires more.*
 
+### With docker-compose:
+```yaml
+version: "3.9"
+
+services:
+  dragonfly: # the new redis
+    image: docker.dragonflydb.io/dragonflydb/dragonfly
+    command:
+      - "--save_schedule=*:*" # save for every 1 minute
+      - "--dbfilename=/data/dump.rdb"
```

Comment:
add `--dir=/data` argument

Reply:
updated

Reply:
You do not need a full path when you pass `--dir` :)

Reply:
Modified.